### PR TITLE
Add build instructions for Ubuntu

### PR DESCRIPTION
**Pull request by kushagharahi**
I know this is kinda a petty PR but I'm not a C developer in my day-to-day so it wasn't apparent to me how to build the binary. So I added some instructions to save the next person 5-15 minutes.

Alternatively it would be cool if a binary was attached to the next and following releases so users don't have to build. 

Thank you very much for this great utility. I tried to outsmart the Kasa app by disconnecting the internet when trying to pair my smart plug to my wifi, but for some asinine reason (data collection) you have to be connected to the internet to pair. 🙄 So this was a great save!!

BTW - Another unsolicited suggestion (sorry): you might get more traffic if you give this repo a different name, it took some creative googling to find this repo. The distinguishing features are the pairing and data collection disabling when comparing it to the other TPLink repos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ reset the plug and try again. Otherwise, the light on your plug will change
 first to blinking blue, then to solid blue indicating that it has successfully
 connected to your AP.
 
+## Build
+
+Ubuntu instructions
+You will need `make` and `build-essential` installed. Then run `make`. This will produce a `hs100` binary which you can use. E.g. `./hs100 192.168.0.1 off`
```
Comment (jkbenaim) [inline README.md]: `build-essential` depends on `make`, so there's no need to install `make` explicitly.

**Comment by jkbenaim** [comment]
These are good suggestions.

Regarding the build instructions, I have one small nitpick that I noted above. I'm not an Ubuntu expert so please check my work on that.

Changing the repo name: in theory I could add "TP-Link" or "Kasa", however I would like to avoid using those trademarks in the repo name. However, I did change the repo's "About" text to include them. It now reads: "CLI tool for TP-Link and Kasa WiFi smart plugs." I hope that helps for visibility.

Please let me know what you think. Thanks!

**EDIT:** regarding shipping a binary, I think that's a good idea too. I'm concerned about differences between distros but I will definitely investigate it.

File: README.md (hunk 1 above)
Comment (kushagharahi) [reply README.md]: Thanks, I had no idea! It's been updated.

**Comment by kushagharahi** [comment]
@jkbenaim "About" updated text sounds good 👍. Totally makes sense why you want to keep trademarks off of the repo name, I had not considered that.

**Comment by jkbenaim** [comment]
LGTM. Thanks! :+1:

**Pull request merged**